ui.apps/src/main/content/jcr_root/apps/core/cif/components/commons/v1/price.html

```diff
@@ -1,18 +1,19 @@
 <!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-  ~ 
+  ~
   ~ Copyright 2020 Adobe. All rights reserved.
   ~ This file is licensed to you under the Apache License, Version 2.0 (the "License");
   ~ you may not use this file except in compliance with the License. You may obtain a copy
   ~ of the License at http://www.apache.org/licenses/LICENSE-2.0
-  ~ 
+  ~
   ~ Unless required by applicable law or agreed to in writing, software distributed under
   ~ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
   ~ OF ANY KIND, either express or implied. See the License for the specific language
   ~ governing permissions and limitations under the License.
-  ~ 
+  ~
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
 <sly data-sly-template.price="${@ priceRange=priceRange, displayYouSave, sku}">
-    <div class="price" data-sly-test="${!priceRange.isEmpty && !priceRange.isRange}" data-product-sku=${sku}>
+  <div data-sly-test=${!priceRange.isMinStartingPrice}>
```

Comment:
This is unnecessary. You can use the existing `<div class="price range"`, just make the "To" part of it conditional and remove it, if it does not exist

```
<span data-sly-test=${!isDiscounted}>${'From' @i18n} ${priceRange.formattedRegularPrice} <sly data-sly-test="${priceRange.regularPriceMax}">${'To' @i18n} ${priceRange.formattedRegularPriceMax}</sly></span>
```

Make sure you handle this for both cases, discounted and not discounted.